go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/docker/cli-docs-tool v0.6.0
 	github.com/docker/docker v24.0.5+incompatible
 	github.com/docker/go-units v0.5.0
+	github.com/dustin/go-humanize v1.0.0
```

Comment:
I'm hesitant to add the dependencies that are added here since the functionality included in this PR is stuff that's also been done in other docker code. My preference is going to be that we utilize those methods rather than add a new way to parse the units.

The same applies to the other two dependencies here.

If there's a compelling reason why this is brand new functionality for docker and these dependencies make implementing that functionality easier, these dependencies seem solid. My preference is just going to be on not adding dependencies if there's an existing alternative.